sql/mob_skills.sql

```diff
@@ -3906,3 +3906,24 @@ INSERT INTO `mob_skills` VALUES (3867,3611,'questionmarks_needles',1,10.0,2000,1
 -- INSERT INTO `mob_skills` VALUES (4078,3822,'#2286',0,7.0,2000,1500,4,0,0,0,0,0,0);
 -- INSERT INTO `mob_skills` VALUES (4079,3823,'#2287',0,7.0,2000,1500,4,0,0,0,0,0,0);
 -- INSERT INTO `mob_skills` VALUES (4080,3824,'#2288',0,7.0,2000,1500,4,0,0,0,0,0,0);
+-- INSERT INTO `mob_skills` VALUES (4080,3824,'#2288',0,7.0,2000,1500,4,0,0,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (4232,549,'potion',0,7.0,2000,1500,4,0,0,0,0,0,0); -- Monberaux
+INSERT INTO `mob_skills` VALUES (4234,550,'x-potion',0,7.0,2000,1500,4,0,0,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (4235,550,'hpyer-potion',0,7.0,2000,1500,4,0,0,0,0,0,0); -- Name is cap'd as "Hpyer Potion"
```

Comment:
[Looks like this got changed in Windower's resources lib in June](https://github.com/Windower/Resources/commit/979184450747ea2efdd03f2f5df0843584eebaeb#diff-06d2dc9b9cf47e05842a393d97098171)

Reply:
![image](https://user-images.githubusercontent.com/1389729/87228067-75b56b80-c3a7-11ea-9589-63f8321cf8cc.png)